README.md

```diff
@@ -1 +1,33 @@
 # eslint-config-sku
+
+This package includes the shareable ESLint configuration used by [sku](https://github.com/seek-oss/sku).
+
+## Usage in sku Projects
+
+The easiest way to use this configuration is with [sku](https://github.com/seek-oss/sku), which includes it by default.
+
+**You don’t need to install it separately in sku projects.**
+
+## Usage Outside of sku
+
+If you want to use this ESLint configuration in a project not built with sku, you can install it with following steps.
+
+First, install this package, ESLint and the necessary plugins.
+
+```sh
+npm install --save-dev eslint-config-sku eslint-import-resolver-node@^0.3.0 eslint-plugin-css-modules@^2.7.1 eslint-plugin-import@^2.2.0 eslint-plugin-react@^7.0.1
```

Comment:
Seeing as all of these dependencies are all listed in the package.json file do we need to suggest installing them separately?